yapapi/__init__.py

```diff
@@ -19,3 +17,19 @@ def enable_default_logger(
     logger.addHandler(handler)
     logger.setLevel(level)
     logger.disabled = False
+
+
+def get_version() -> str:
+    pyproject_path = Path(__file__).parents[1] / "pyproject.toml"
+    if pyproject_path.exists():
+        import toml
```

Comment:
Do we have a good reason to have this `import` here instead of at the top of the file (as PEP08 requires)?

Reply:
> Do we have a good reason to have this `import` here instead of at the top of the file (as PEP 8 requires)?

yeah, I was reluctant to add it as a non-dev dependency but realized that it'd have to be added anyway - to prevent a case when someone issues a `poetry install --no-dev` from source and ends up in a non-functional library ...